vms/platformvm/service_test.go

```diff
@@ -1107,3 +1108,65 @@ func TestServiceGetSubnets(t *testing.T) {
 		},
 	}, response.Subnets)
 }
+
+func TestGetFeeConfig(t *testing.T) {
+	tests := []struct {
+		name     string
+		etnaTime time.Time
+		expected fee.Config
+	}{
+		{
+			name:     "pre-etna",
+			etnaTime: time.Now().Add(time.Hour),
+			expected: fee.Config{},
+		},
+		{
+			name:     "post-etna",
+			etnaTime: time.Now().Add(-time.Hour),
+			expected: defaultDynamicFeeConfig,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			require := require.New(t)
+
+			service, _, _ := defaultService(t)
+			service.vm.Config.UpgradeConfig.EtnaTime = test.etnaTime
+
+			var reply fee.Config
+			require.NoError(service.GetFeeConfig(nil, nil, &reply))
+			require.Equal(test.expected, reply)
+		})
+	}
+}
+
+func TestGetFeeState(t *testing.T) {
+	require := require.New(t)
+
+	seed := time.Now().UnixNano()
+	t.Logf("Seed: %d", seed)
+	random := rand.New(rand.NewSource(seed)) // #nosec G404
```

Comment:
This suggests that this test should be `func FuzzGetFeeState(f *testing.F)` with the internal test func remaining the same as below.

Reply:
ok